Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIScriptingCocoa.mm

```diff
@@ -53,17 +57,91 @@
 static NSString * const worldKey = @"world";
 
 static NSString * const  cssKey = @"css";
-// FIXME: <https://webkit.org/b/261765> Consider adding support for cssOrigin.
 
 static NSString * const mainWorld = @"MAIN";
 static NSString * const isolatedWorld = @"ISOLATED";
 
 static int64_t const mainFrameID = 0;
```

Comment:
your static size_t mainFrameID defined in WebExtensionDynamicScripts.h might cause problems with thIs.

Reply:
Yeah this should be removed too.